code/game/machinery/Sleeper.dm

```diff
@@ -473,6 +473,21 @@
 	updateUsrDialog()
 	return
 
+
+/obj/machinery/sleeper/upgraded
+	name = "advanced Sleeper"
```

Comment:
advanced capitalization 🤔

Reply:
body scanner -> advanced bodyscanner
\improper Sleeper -> advanced Sleeper

Reply:
This entire capitalization must beb purged

Reply:
ok